assets/js/reviewerAction.js

```diff
@@ -0,0 +1,30 @@
+$(function () {
+  const form = document.querySelector('form')
+  form.addEventListener('submit', e => {
+    e.preventDefault()
```

Comment:
Doesn't this stop the submit from working?

Could we have an empty callback and not have to `preventDefault`?